config/locales/en.yml

```diff
@@ -266,8 +266,8 @@ en:
 
           When you are ready to upgrade your Rancher instance, please see the [Rancher docs](https://docs.ranchermanager.rancher.io/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades)
           for details.
-        failed_caption: "**Deployment failed. Something went wrong during the deployment.**"
-        failed_info: "The resources created should be cleaned up, if any. Retry another deployment once the issue has been resolved."
+        failed_caption: "**Cleaning up the resources failed. Something went wrong.**"
```

Comment:
Same issue - translation keys can't just be repurposed - you need to add extra translations specific to the cleanup failure.

Reply:
Fixed in [38b8a60](https://github.com/SUSE-Enceladus/suse-rancher-setup/pull/168/commits/38b8a60315b33dd62b92cc2202ede0a6bdb9215e)